test/framework/gomega_utils.go

```diff
@@ -12,16 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package plants
+package framework
 
-import (
-	"testing"
+import "github.com/onsi/gomega"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-func TestPlant(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Plant Test Suite")
+// ExpectNoError checks if an error has occurred
+func ExpectNoError(err error) {
```

Comment:
Make the function signature

```go
func ExpectNoError(actual interface{}, extra ...interface{}) 
```

So instead of 

```go
_, err := someFunc()
ExpectNoError(err)
```

You can do 

```go
ExpectNoError(someFunc())
```